mozilla/foundation.mozilla.org
# Remove moment

Closes #7310 


**Steps to test**:

1. Visit https://foundation-s-remove-mom-q4bh4f.mofostaging.net/en/news/
2. The dates for the publications should look the same as before, rendered as a full month name and year:
<img width="872" alt="Screen Shot 2022-02-22 at 3 56 57 PM" src="https://user-images.githubusercontent.com/18314510/155239888-0bd8fd8c-8d8a-4969-b3b6-b37ab331b91c.png">
3. If everything is looking as it should, testing is complete!
